.env

```diff
@@ -1,6 +1,7 @@
 VUE_APP_AUTH_URL=https://auth.endpass.com
 VUE_APP_OAUTH_SERVER=https://api.endpass.com/v1
 VUE_APP_OAUTH_CLIENT_ID=ee8b2a12-089d-45a2-a224-89eab10fa96d
+VUE_APP_DEMO_SERVER=https://endpass.github.io/connect-demo
```

Comment:
where this env is used? and for what?

Reply:
earlier was used for determine current server

Reply:
remove this, not used variable